ledger/eval_test.go

```diff
@@ -49,6 +51,14 @@ func init() {
 	minFee = basics.MicroAlgos{Raw: params.MinTxnFee}
 }
 
+func gadify(txgroup []transactions.SignedTxn) []transactions.SignedTxnWithAD {
+	txgroupad := make([]transactions.SignedTxnWithAD, len(txgroup))
+	for i, tx := range txgroup {
+		txgroupad[i].SignedTxn = tx
+	}
+	return txgroupad
+}
+
```

Comment:
This method does not belong here. In fact, it should be placed (maybe) in bookkeeping, as the same logic is also used in `addToPendingBlockEvaluatorOnce` : 
```golang
	txgroupad := make([]transactions.SignedTxnWithAD, len(txgroup))
	for i, tx := range txgroup {
		txgroupad[i].SignedTxn = tx
	}
```